Apigee/SmartDocs/Model.php

```diff
@@ -208,7 +208,7 @@ public static function fromArray(Model $model, array $array)
         foreach ($array as $key => $value) {
             if (property_exists($model, $key)) {
                 if ($key == 'activeRevision' && is_array($value) && !empty($value)) {
-                    $revision = new Revision($model->getConfig(), $model->getUuid());
+                    $revision = new Revision($model->getConfig(), $model->getUuid(), $model->getName());
```

Comment:
The new endpoint is https://:host/v1/organizations/:org/apimodels/:model/import/file?format=swagger, where :model is the name of the apimodel.